### ISSUE TITLE
Support Python 3.9

### ISSUE THREAD

**Issue by l0b0** [aws/aws-lambda-base-images#11]
Python 3.9 was [considered stable](https://www.python.org/downloads/release/python-390/) on 2020-10-05.

**Comment by mat100payette** [comment]
Is there really no rough estimate as to when this could be possible ?

**Comment by l0b0** [comment]
[Looks like](https://aws.amazon.com/about-aws/whats-new/2021/08/aws-lambda-adds-support-python-3-9/) this is [done](https://docs.aws.amazon.com/lambda/latest/dg/python-image.html). Thank you team!

**Comment by tsuga** [comment]
It is here https://github.com/aws/aws-lambda-base-images/tree/python3.9

**Comment by alonbl** [comment]
yes, it is done but same latency is for python-3.10[1], we all know that it is not that difficult to migrate.

[1] https://github.com/aws/aws-lambda-base-images/issues/31
